package.json

```diff
@@ -1,25 +1,25 @@
 {
   "name": "@ottofeller/eslint-config-ottofeller",
-  "version": "0.9.1",
+  "version": "0.10.1",
```

Comment:
@quesabe let's bump to `0.10.0`, not keeping the patch-part from the previous version.